### PR TITLE
Add capability to show if group sharing is enabled

### LINKED ISSUE #23477: Expose disabled group sharing in the capabilities API
Now that the feature to disable group sharing is in (https://github.com/owncloud/core/pull/23398) we should offer this information to our clients via the capabilities API. So they can adjust their wording slightly.

Just for consitency.

### PR DESCRIPTION
**Pull request by rullzer**
fixes #23477

Now clients can correct wording to properly reflect who a user can share with.

CC: @MorrisJobke @PVince81 @nickvergessen

### CODE REVIEW

**Comment by nickvergessen** [comment]
We have integration tests for capabilities :dancers:

**Comment by rullzer** [comment]
Ah right... fixed...

File: build/integration/capabilities_features/capabilities.feature [hunk 1]
```diff
@@ -212,6 +221,27 @@ Feature: capabilities
...
+			| files_sharing | api_enabled | 1 |
+			| files_sharing | public@@@enabled | 1 |
+			| files_sharing | public@@@upload | 1 |
+			| files_sharing | resharing | 1 |
+			| files_sharing | federation@@@outgoing | 1 |
+			| files_sharing | federation@@@incoming | 1 |
+			| files_sharing | group_sharing         | EMPTY |
+			| files | bigfilechunking | 1 |
+			| files | undelete | 1 |
+			| files | versioning | 1 |
```
Comment (nickvergessen) [inline build/integration/capabilities_features/capabilities.feature]: new line

Comment (rullzer) [inline build/integration/capabilities_features/capabilities.feature]: fixed

**Comment by nickvergessen** [comment]
:+1: maybe @schiesbn can review as well?

**Comment by MorrisJobke** [comment]
Mysql failed:

```
06:16:31 There was 1 failure:
06:16:31 
06:16:31 1) OCA\Files_sharing\Tests\EtagPropagation::testOwnerWritesToSingleFileShare
06:16:31 Failed asserting that the etag for "" of user test-share-user2 has changed
06:16:31 Failed asserting that '066a092a11182ce38de3e69f3e5c9948' is not equal to <string:066a092a11182ce38de3e69f3e5c9948>.
06:16:31 
06:16:31 /ssd/jenkins/workspace/core-ci-linux@4/database/mysql/label/SLAVE/apps/files_sharing/tests/propagationtestcase.php:63
06:16:31 /ssd/jenkins/workspace/core-ci-linux@4/database/mysql/label/SLAVE/apps/files_sharing/tests/etagpropagation.php:131
06:16:31
```

**Comment by rullzer** [comment]
@MorrisJobke seems like a mysql hickup... (also since this does not toch any of the etag code :wink: )

**Comment by MorrisJobke** [comment]
Tested and works :+1:

**Pull request merged**